.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Report a bug encountered while operating KUDO
```

Comment:
I would have intuitively put quotes around this, but do not have a strong argument for or against